packages/angular_devkit/core/src/json/schema/visitor.ts

```diff
@@ -69,7 +69,7 @@ function _visitJsonRecursive<ContextT>(
       ? value as Observable<JsonValue>
       : observableOf(value as JsonValue)
   ).pipe(
-    concatMap((value: JsonValue) => {
+    concatMap((value): Observable<JsonValue> => {
```

Comment:
That’s not valid TypeScript is it?

Reply:
It's missing the input type, or doesn't it need it?

Reply:
It's what Rado recommended in these situations. Unfortunately, TS can't infer the return in these cases where there's branching code.

Reply:
The input type is inferred.